econplayground/main/models.py

```diff
@@ -40,6 +40,7 @@ class Cohort(models.Model):
     password = models.CharField(max_length=256, null=True, blank=True)
 
     instructors = models.ManyToManyField(User)
+    is_sample = models.BooleanField(null=True, blank=True, unique=True)
```

Comment:
another option here: default=False

Reply:
definitely.. but this is actually intentional. Leaving the default as None, and adding unique=True allows there to only ever be one Cohort whose is_sample is True. Using default=False would fail with a UniqueAttributeError. I will add a comment here, because it's not that clear.